forge-game/src/main/java/forge/game/spellability/SpellAbility.java

```diff
@@ -149,6 +149,7 @@ public static class EmptySa extends SpellAbility {
     protected ApiType api = null;
 
     private List<Mana> payingMana = Lists.newArrayList();
+    @Deprecated
     private List<SpellAbility> paidAbilities = Lists.newArrayList();
```

Comment:
if it is deprecated, can it be removed too?

Reply:
Hmn, I thought it wasn't used anywhere but actually refunding does